test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1

```diff
@@ -803,6 +886,23 @@ Describe "Invoke-WebRequest tests" -Tags "Feature" {
         $response.Content.UserAgent | Should Match $Pattern
     }
 
+    It "Verifies Invoke-WebRequest Certificate Authentication Fails without -Certificate"{
+        $command = 'Invoke-WebRequest https://prod.idrix.eu/secure/'
```

Comment:
@iSazonov  I think you meant [This](https://www.appveyor.com/docs/services-databases/#internet-information-services). There would probably have to be a separate solution for AppVeyor and [Travis CI](https://docs.travis-ci.com/user/gui-and-headless-browsers/#Starting-a-Web-Server). It would probably be best if there was some kind of cross-platform solution that worked regardless of CI environment.